src/JoomlaBrowser.php

```diff
@@ -500,6 +506,14 @@ public function selectMultipleOptionsInChosen($label, $options)
     public function doAdministratorLogout()
     {
         $I = $this;
+
+        if($this->isPhone())
+        {
+            $I->click(['xpath' => "//div[@class='navbar-inner']//div[@class='container-fluid']//a[@class='btn btn-navbar collapsed']"]);
```

Comment:
Can't that be simplified? This xpath is way too long. You have multiple ways adressing an element, from CSS, over text etc.